include/hip/hcc_detail/hip_atomic.h

```diff
@@ -73,6 +73,14 @@ float atomicAdd(float* address, float val)
 
     return __uint_as_float(r);
 }
+
+__device__
+inline
+void atomicAddNoRet(float* address, float val)
+{
+    __ockl_global_atomic_add_f32(address, val);
```

Comment:
The OCKL function requires an address_space(1) pointer for the first argument.  Will need a cast if address here.

Reply:
fixed via b87c508